calico-cloud/image-assurance/configuring-the-runtime-view.mdx

```diff
@@ -0,0 +1,21 @@
+---
+description: Use the Runtime View to find vulnerabilities in your cluster.
+---
+
+# Configuring the Runtime View
+
+**Enable Runtime View**
```

Comment:
Need to show / describe how to enable the run time view, not just what happens when you do.